tests/OSPSuite.R.Tests/Services/ConcurrentSimulationRunnerSpecs.cs

```diff
@@ -87,6 +88,64 @@ public void should_complete_without_error()
       }
    }
 
+   public class When_running_a_batch_simulation_with_exception : concern_for_ConcurrentSimulationRunner
+   {
+      private ConcurrentRunSimulationBatch _concurrentRunSimulationBatch;
+      private ConcurrencyManagerResult<SimulationResults>[] _results;
+      private IModelCoreSimulation _simulation;
+      private readonly List<string> _ids = new List<string>();
+      private readonly List<SimulationBatchRunValues> _simulationBatchRunValues = new List<SimulationBatchRunValues>();
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         _simulation = _simulationPersister.LoadSimulation(HelperForSpecs.DataFile("S1.pkml"));
+
+         // Force an error during simulation run
+         _simulation.SimulationSettings.Solver.MxStep = 3;
+         _concurrentRunSimulationBatch = new ConcurrentRunSimulationBatch
+         (
+            _simulation,
+            new SimulationBatchOptions
+            {
+               VariableMolecules = new[]
+               {
+                  new[] { "Organism", "Kidney", "Intracellular", "Caffeine" }.ToPathString()
+               },
+
+               VariableParameters = new[]
+               {
+                  new[] { "Organism", "Liver", "Volume" }.ToPathString(),
+                  new[] { "Organism", "Hematocrit" }.ToPathString(),
+               }
+            }
+         );
+
+         sut.AddSimulationBatch(_concurrentRunSimulationBatch);
+      }
+
+      protected override void Because()
+      {
+         
+         _simulationBatchRunValues.Add(new SimulationBatchRunValues
+         {
+            InitialValues = new[] { 10.0 },
+            ParameterValues = new[] { 3.5, 0.53 }
+         });
+
+         _concurrentRunSimulationBatch.AddSimulationBatchRunValues(_simulationBatchRunValues[0]);
+
+         _ids.AddRange(_concurrentRunSimulationBatch.SimulationBatchRunValues.Select(x => x.Id));
+         _results = sut.RunConcurrently();
+      }
+
+      [Observation]
+      public void should_return_a_result_with_success_set_to_false()
+      {
+         _results.All(x => x.Succeeded).ShouldBeFalse();
```

Comment:
Do we have a test that now verifies that this flag is sometimes true?  (e.g. can you set the parameter for one run instead as for all). Then one run should be success and the one where it fails should be false.

Reply:
We do now.